pkg/resource/generator.go

```diff
@@ -147,14 +147,19 @@ func (g *Generator) syncStorage(cr *imageregistryv1.Config) error {
 		if !exists {
 			runCreate = true
 		}
+
+		// GCP: If during previous reconcile error was set, failing to add required
+		// tags, setting runCreate to true  to retry adding the tags.
+		if cr.Spec.Storage.GCS != nil {
+			runCreate = true
+		}
 	}
 
 	if runCreate {
-		reconf := g.storageReconfigured(cr, g.kubeconfig, g.listers)
 		if err := driver.CreateStorage(cr); err != nil {
 			return err
 		}
-		if reconf {
+		if g.storageReconfigured(cr, g.kubeconfig, g.listers) {
```

Comment:
is there a specific reason for this move?

Reply:
While making the code changes, IDE suggested the change :)

Reply:
suggestion incorporated.

Reply:
did you forget to push @bharath-b-rh ? code looks the same

Reply:
Sorry, yes I had missed committing changes and commented as done. It's updated now.